keras_cv/models/object_detection/yolo_v8/yolo_v8_detector_test.py

```diff
@@ -171,21 +184,21 @@ def test_update_prediction_decoder(self):
             backbone=keras_cv.models.YOLOV8Backbone.from_preset(
                 "yolo_v8_s_backbone"
             ),
-            prediction_decoder=keras_cv.layers.MultiClassNonMaxSuppression(
+            prediction_decoder=keras_cv.layers.NonMaxSuppression(
```

Comment:
Slightly different, because this runs global NMS instead of separating by class. We can support multi-class NMS down the road for all backends but for now it wasn't tractable in the time we had.

Reply:
Won't it be confusing for tf.keras users to get different behavior? And isn't this one a lot slower if I recall our discussions?

Reply:
All users will get the same `NonMaxSuppression` layer here, and `tf.keras` users can elect to use `MultiClassNonMaxSuppression`.

After implementing a native version for TF and Torch, it's no longer slower